### ISSUE TITLE
Blank media type throws error (valid epub)

### ISSUE THREAD

**Issue by nruffilo** [vers-one/EpubReader#47]
I have a valid epub, but the content.OPF file contains an item with a blank media type:
<item id="A_process.log" href="process.log" media-type=""/>

Sadly I have to be able to process files that pass epub check, and this does.  Below is the stack trace from the fail.  Ideally this would gracefully just ignore the file.  Sadly I cannot provide the file as it contains copyrighted material, but I believe just adding the the file and a blank media-type to any should produce the same issue.

Thank you!

   at VersOne.Epub.Internal.PackageReader.ReadManifest(XElement manifestNode)
   at VersOne.Epub.Internal.PackageReader.<ReadPackageAsync>d__0.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at VersOne.Epub.Internal.SchemaReader.<ReadSchemaAsync>d__0.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at VersOne.Epub.EpubReader.<OpenBookAsync>d__10.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at VersOne.Epub.EpubReader.<ReadBookAsync>d__9.MoveNext()

**Comment by vers-one** [comment]
I added an option to skip invalid manifest items.

Here's how you can use it:
```
EpubReaderOptions options = new()
{
    PackageReaderOptions = new PackageReaderOptions()
    {
        SkipInvalidManifestItems = true
    }
};
```

Then you can pass the `options` as the second argument to one of the `EpubReader`'s `OpenBook` / `OpenBookAsync` / `ReadBook` / `ReadBookAsync` methods.

Could you check if this solution works for you before I publish it to Nuget?

PS. Keep in mind that this solution may cause issues at a later stage if the invalid manifest item is referenced from somewhere else (e.g. if it is the cover or the table of contents).

**Comment by vers-one** [comment]
Implemented in v3.2. Here's the [documentation](http://localhost:8080/malformed-epub/index.html#invalid-epub-manifest-items) for the new option.
